src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -459,6 +461,7 @@ As part of your monitoring, make sure to keep an eye on your machine's performan
 
 ## Further reading {#further-reading}
 
+- [Ethereum Staking Guides](https://github.com/SomerEsat/ethereum-staking-guides) - _Somer Esat, updated often_
```

Comment:
I added these staking guides because they are just good and up to date resource on running node, various client combos. Maybe I would change the title to 'Running node and staking', I want people who don't want to stake to look at it as well